src/visualization/Turi Create Visualization/src/user_interface/package.json

```diff
@@ -12,6 +12,8 @@
         "react": "^16.2.0",
         "react-dom": "^16.2.0",
         "react-json-pretty": "^1.7.5",
+        "react-vega": "^4.0.2",
```

Comment:
Make sure the react-vega version works well with the vega version we are using. And please pin the version explicitly so we know what we're shipping is what's been tested (no `^`, which allows npm to pick up a later version).